column_transforms/rasgo_concat/rasgo_concat.sql

```diff
@@ -0,0 +1,8 @@
+SELECT
+*
+, CONCAT(
+    {%- for obj in concat_list %}
+    {{obj}}{{ ", " if not loop.last else "" }}
+    {%- endfor %}
+)
```

Comment:
Snowflake automatically creates an alias column name based on the arguments passed to the function. For instance, the sample code in the YAML file:
```source.transform(
    source_code=s2,
    concat_list=["DS_WEATHER_ICON", "'some_str'", "'_5'"],
).preview_sql()```

produces an extra column named `CONCAT( DS_WEATHER_ICON, 'SOME_STR', '_5' )`. If this is cumbersome or crappy, I can just create an alias that we want using the arguments here.

Reply:
let's use cleanse_name and make the alias ourselves. i believe the snowflake column naming would have to be quoted which would break the rest of rasgo

Reply:
I added `cleanse_name` each place we create an alias. Lemme know what u think